src/main/scala/mesosphere/marathon/core/launchqueue/impl/RateLimiter.scala

```diff
@@ -69,10 +77,16 @@ private[launchqueue] class RateLimiter(clock: Clock) {
 }
 
 private object RateLimiter {
+  /**
+    * The viability threshold may be tuned later, once the mechanism
+    * has been validated.
+    */
+  private val minimumTaskExecutionSeconds: Long = 60
```

Comment:
Oh, it's even better if the parameter can be tuned through command line flags .. What do you think about not having this parameter hidden ?